### ISSUE TITLE
train.py

### ISSUE THREAD

**Issue by jedota** [thangvubk/FEQE#13]
Hi!
In the training section Number 2

Finetune with MSE loss on scale 4 (FEQE-P): 
python main.py --checkpoint checkpoint/mse_s4 --alpha_vgg 0 --pretrained_model checkpoint_test/mse_s2/model.ckpt

The main.py is not available or are you mean train.py
